qwik-graphql-tailwind/src/components/user-repos/user-repos.tsx

```diff
@@ -30,32 +26,30 @@ export const UserRepos = component$(({ repos, owner }: UserReposProps) => {
     track(() => filters.language);
     track(() => filters.sortBy);
     track(() => filters.type);
-    state.searchResponse = repos.nodes;
+    track(() => location.query.after);
+    track(() => location.query.before);
+
     if (filters.search) {
-      state.searchResponse = repoDataFilteredBySearch(filters?.search || '', state.searchResponse);
+      repos.nodes = repoDataFilteredBySearch(filters?.search || '', repos.nodes);
     }
 
     if (filters.language) {
-      state.searchResponse = repoDataFilteredByLanguage(filters?.language, state.searchResponse);
+      repos.nodes = repoDataFilteredByLanguage(filters?.language, repos.nodes);
     }
 
     if (filters.type) {
-      state.searchResponse = repoDataFilteredByType(filters.type, state.searchResponse);
+      repos.nodes = repoDataFilteredByType(filters.type, repos.nodes);
     }
 
     if (filters.sortBy) {
-      state.searchResponse = sortedRepoData(filters.sortBy, state.searchResponse);
+      repos.nodes = sortedRepoData(filters.sortBy, repos.nodes);
     }
   });
 
```

Comment:
Please, lets first reverse to what was there before.
the filter and sort were all working. But now they are not

Reply:
![Screenshot 2023-01-12 at 12 59 04](https://user-images.githubusercontent.com/28502531/212061203-80ac54a8-6e54-4b9c-b3ce-3e37f3a50b97.jpg)

Another example...